### PR TITLE
Fixes images on Worker service bindings page

### LINKED ISSUE #4367: All images on Workers Service Bindings page are broken
### Example URL(s)

https://developers.cloudflare.com/workers/platform/bindings/about-service-bindings

### Actual Behavior

The images are broken

### Expected Behavior

The images are displayed

### Additional information

image

### PR DESCRIPTION
**Pull request by jrf0110**
Closes #4367

### CODE REVIEW

**Review by deadlypants1973 — APPROVED** [review]
Thank you very much!

File: content/workers/platform/bindings/about-service-bindings.md [hunk 1]
```diff
@@ -29,11 +29,11 @@ To manage a Workers Service binding:
 
 You can also change the environment of a Workers Service binding, so you can target a specific version of a Workers Service.
 
-![To configure a Service binding, go to your Worker > Settings > Variables and follow the steps above](../media/service-bindings-config.png)
+![To configure a Service binding, go to your Worker > Settings > Variables and follow the steps above](./media/service-bindings-config.png)
 
 Workers bound to your Worker will be listed in [**Workers**](https://dash.cloudflare.com/?zone=workers) > your **Worker** > **Triggers** > **Bound Services**. Your team can easily view cross-service dependencies in this manner.
 
-![Your team can view cross-service dependencies in the Cloudflare dashboard Account Home > Workers > Triggers](../media/service-bindings-triggers.png)
+![Your team can view cross-service dependencies in the Cloudflare dashboard Account Home > Workers > Triggers]../media/service-bindings-triggers.png)
```
Comment (deadlypants1973) [inline content/workers/platform/bindings/about-service-bindings.md]:
```suggestion
![Your team can view cross-service dependencies in the Cloudflare dashboard Account Home > Workers > Triggers](./media/service-bindings-triggers.png)
```

File: content/workers/platform/bindings/about-service-bindings.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Service bindings allow you to:
 
 While the interface among Service bindings is HTTP, the networking is not. Unlike the typical microservice architecture, where services communicate over a network and can suffer from latency or interruption, Service bindings are a zero-cost abstraction. When one Worker invokes another, there is no network delay and the request is executed immediately.
 
-![Service bindings are a zero-cost abstraction](../media/service-bindings-comparison.png)
+![Service bindings are a zero-cost abstraction](./media/service-bindings-comparison.png)
```
Comment (WalshyDev) [inline content/workers/platform/bindings/about-service-bindings.md]:
```suggestion
![Service bindings are a zero-cost abstraction](/workers/platform/bindings/media/service-bindings-comparison.png)
```

File: content/workers/platform/bindings/about-service-bindings.md [hunk 3]
```diff
@@ -29,11 +29,11 @@ To manage a Workers Service binding:
 
 You can also change the environment of a Workers Service binding, so you can target a specific version of a Workers Service.
 
-![To configure a Service binding, go to your Worker > Settings > Variables and follow the steps above](../media/service-bindings-config.png)
+![To configure a Service binding, go to your Worker > Settings > Variables and follow the steps above](./media/service-bindings-config.png)
```
Comment (WalshyDev) [inline content/workers/platform/bindings/about-service-bindings.md]:
```suggestion
![To configure a Service binding, go to your Worker > Settings > Variables and follow the steps above](/workers/platform/bindings/service-bindings-config.png)
```

File: content/workers/platform/bindings/about-service-bindings.md [hunk 4]
```diff
@@ -29,11 +29,11 @@ To manage a Workers Service binding:
...
 
 Workers bound to your Worker will be listed in [**Workers**](https://dash.cloudflare.com/?zone=workers) > your **Worker** > **Triggers** > **Bound Services**. Your team can easily view cross-service dependencies in this manner.
 
-![Your team can view cross-service dependencies in the Cloudflare dashboard Account Home > Workers > Triggers](../media/service-bindings-triggers.png)
+![Your team can view cross-service dependencies in the Cloudflare dashboard Account Home > Workers > Triggers](./media/service-bindings-triggers.png)
```
Comment (WalshyDev) [inline content/workers/platform/bindings/about-service-bindings.md]:
```suggestion
![Your team can view cross-service dependencies in the Cloudflare dashboard Account Home > Workers > Triggers](/workers/platform/bindings/service-bindings-triggers.png)
```

File: content/workers/platform/bindings/about-service-bindings.md [hunk 5]
```diff
@@ -45,7 +45,7 @@ First, you will create both a `gateway` and `auth` Worker. Once they are set up,
 
 To manage Service bindings, go to Account Home > [**Workers**](https://dash.cloudflare.com/?zone=workers) > your **Worker** > **Settings**> **Variables** > **Service bindings** > **Edit variables**.
 
-![Selecting Edit variables to create new bindings and edit existing bindings that enable Worker-to-Worker communication](../media/service-bindings.png)
+![Selecting Edit variables to create new bindings and edit existing bindings that enable Worker-to-Worker communication](./media/service-bindings.png)
```
Comment (WalshyDev) [inline content/workers/platform/bindings/about-service-bindings.md]:
```suggestion
![Selecting Edit variables to create new bindings and edit existing bindings that enable Worker-to-Worker communication](/workers/platform/bindings/service-bindings.png)
```

**Review by WalshyDev — COMMENTED** [review]
Absolute all the way :^)

File: content/workers/platform/bindings/about-service-bindings.md [hunk 6]
```diff
@@ -29,11 +29,11 @@ To manage a Workers Service binding:
 
 You can also change the environment of a Workers Service binding, so you can target a specific version of a Workers Service.
 
-![To configure a Service binding, go to your Worker > Settings > Variables and follow the steps above](../media/service-bindings-config.png)
+![To configure a Service binding, go to your Worker > Settings > Variables and follow the steps above](/workers/platform/bindings/service-bindings-config.png)
```
Comment (WalshyDev) [inline content/workers/platform/bindings/about-service-bindings.md]:
```suggestion
![To configure a Service binding, go to your Worker > Settings > Variables and follow the steps above](/workers/platform/bindings/media/service-bindings-config.png)
```

File: content/workers/platform/bindings/about-service-bindings.md [hunk 7]
```diff
@@ -45,7 +45,7 @@ First, you will create both a `gateway` and `auth` Worker. Once they are set up,
 
 To manage Service bindings, go to Account Home > [**Workers**](https://dash.cloudflare.com/?zone=workers) > your **Worker** > **Settings**> **Variables** > **Service bindings** > **Edit variables**.
 
-![Selecting Edit variables to create new bindings and edit existing bindings that enable Worker-to-Worker communication](../media/service-bindings.png)
+![Selecting Edit variables to create new bindings and edit existing bindings that enable Worker-to-Worker communication](/workers/platform/bindings/service-bindings.png)
```
Comment (WalshyDev) [inline content/workers/platform/bindings/about-service-bindings.md]:
```suggestion
![Selecting Edit variables to create new bindings and edit existing bindings that enable Worker-to-Worker communication](/workers/platform/bindings/media/service-bindings.png)
```

**Review by WalshyDev — COMMENTED** [review]
I cannot copy paste correctly apparently 🤦

File: content/workers/platform/bindings/about-service-bindings.md [hunk 8]
```diff
@@ -29,11 +29,11 @@ To manage a Workers Service binding:
 
 You can also change the environment of a Workers Service binding, so you can target a specific version of a Workers Service.
 
-![To configure a Service binding, go to your Worker > Settings > Variables and follow the steps above](../media/service-bindings-config.png)
+![To configure a Service binding, go to your Worker > Settings > Variables and follow the steps above](/workers/platform/bindings/service-bindings-config.png)
 
 Workers bound to your Worker will be listed in [**Workers**](https://dash.cloudflare.com/?zone=workers) > your **Worker** > **Triggers** > **Bound Services**. Your team can easily view cross-service dependencies in this manner.
 
-![Your team can view cross-service dependencies in the Cloudflare dashboard Account Home > Workers > Triggers](../media/service-bindings-triggers.png)
+![Your team can view cross-service dependencies in the Cloudflare dashboard Account Home > Workers > Triggers](/workers/platform/bindings/service-bindings-triggers.png)
```
Comment (WalshyDev) [inline content/workers/platform/bindings/about-service-bindings.md]:
```suggestion
![Your team can view cross-service dependencies in the Cloudflare dashboard Account Home > Workers > Triggers](/workers/platform/bindings/media/service-bindings-triggers.png)
```

**Pull request merged**